crates/runtime/src/component/dataset.rs

```diff
@@ -341,398 +348,7 @@ impl Dataset {
     }
 }
 
-pub mod acceleration {
```

Comment:
I moved the acceleration module into its own file, as it was getting quite large. I also wanted two sub-modules for acceleration